src/pages/Privacy.vue

```diff
@@ -863,92 +863,34 @@
         einer anderen natürlichen oder juristischen Person erforderlich ist.
       </p>
       <p>
-        Weitere Hinweise zu der Löschung von personenbezogenen Daten können
-        ferner im Rahmen der einzelnen Datenschutzhinweise dieser
-        Datenschutzerklärung erfolgen.
+        {{ $t('imprintContact1')
+        }}<a href="mailto:info@coronahelfer.eu"> info@coronahelfer.eu</a>
       </p>
-      <h5 id="m15">Änderung und Aktualisierung der Datenschutzerklärung</h5>
+
+      <h3 class="uppercase">{{ $t('imprintSMTitle') }}</h3>
+      <p>{{ $t('imprintSM1') }}</p>
       <p>
-        Wir bitten Sie, sich regelmäßig über den Inhalt unserer
-        Datenschutzerklärung zu informieren. Wir passen die
-        Datenschutzerklärung an, sobald die Änderungen der von uns
-        durchgeführten Datenverarbeitungen dies erforderlich machen. Wir
-        informieren Sie, sobald durch die Änderungen eine
-        Mitwirkungshandlung Ihrerseits (z.B. Einwilligung) oder eine
-        sonstige individuelle Benachrichtigung erforderlich wird.
+        Facebook:
+        <a href="https://www.facebook.com/coronahelfer.eu/"
+          >https://www.facebook.com/coronahelfer.eu/</a
+        >
       </p>
       <p>
-        Sofern wir in dieser Datenschutzerklärung Adressen und
-        Kontaktinformationen von Unternehmen und Organisationen angeben,
-        bitten wir zu beachten, dass die Adressen sich über die Zeit ändern
-        können und bitten die Angaben vor Kontaktaufnahme zu prüfen.
+        Instagram:
+        <a href="https://www.instagram.com/coronahelfer.eu/"
```

Comment:
Please tidy up your html tags for all those links and don't show the link, instead the name of the social network